### PR TITLE
Add content/es/docs/reference/glossary/pod-priority

### PR DESCRIPTION
**Pull request by krlooss**
created file content/es/docs/reference/glossary/pod-priority.md to translate Pod Priority to Spanish

### CODE REVIEW

**Comment by alexbrand** [comment]
Hey @krlooss, thanks for the PR. Do you mind updating the PR title and providing a description of this PR? Thanks!

**Comment by krlooss** [comment]
> Hey @krlooss, thanks for the PR. Do you mind updating the PR title and providing a description of this PR? Thanks!

Done!

File: content/es/docs/reference/glossary/pod-priority.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- operation
+---
+ Pod Priority indica la importancia de un Pod con relación a otros Pods.
+
+<!--more-->
+
+[Pod Priority](/docs/concepts/configuration/pod-priority-preemption/#pod-priority) da la habilidad de configurar prioridades del programador de un Pod más altas o bajas que otros Pods - lo cual es una característica importante para Clusters Workload en producción.
```
Comment (raelga) [inline content/es/docs/reference/glossary/pod-priority.md]:
```suggestion
[Pod Priority](/docs/concepts/configuration/pod-priority-preemption/#pod-priority) da la habilidad de configurar prioridades del programador de un {{< glossary_tooltip text="Pod" term_id="pod" >}} más altas o bajas que otros {{< glossary_tooltip text="Pods" term_id="pod" >}} - lo cual es una característica importante para cargas de trabajo en producción. 
```

File: content/es/docs/reference/glossary/pod-priority.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+date: 2019-01-31
+full_link: /docs/concepts/configuration/pod-priority-preemption/#pod-priority
+short_description: >
+  Pod Priority indica la importancia de un Pod con relación a otros Pods. 
+
+aka:
+tags:
+- operation
+---
+ Pod Priority indica la importancia de un Pod con relación a otros Pods.
```
Comment (raelga) [inline content/es/docs/reference/glossary/pod-priority.md]:
```suggestion
 Pod Priority indica la importancia de un {{< glossary_tooltip text="Pod" term_id="pod" >}} con relación a otros {{< glossary_tooltip text="Pods" term_id="pod" >}}.
```

File: content/es/docs/reference/glossary/pod-priority.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Pod Priority
+id: pod-priority
+date: 2019-01-31
+full_link: /docs/concepts/configuration/pod-priority-preemption/#pod-priority
+short_description: >
+  Pod Priority indica la importancia de un Pod con relación a otros Pods.
```
Comment (raelga) [inline content/es/docs/reference/glossary/pod-priority.md]:
```suggestion
  Pod Priority indica la importancia de un {{< glossary_tooltip text="Pod" term_id="pod" >}} con relación a otros {{< glossary_tooltip text="Pods" term_id="pod" >}}. 
```

File: content/es/docs/reference/glossary/pod-priority.md (hunk 2 above)
Comment (krlooss) [reply content/es/docs/reference/glossary/pod-priority.md]: accepted the one below

**Comment by raelga** [comment]
Muchas gracias!

/approve

**Pull request merged**